index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="wrapper">
+        <header class="main-nav">
+            <nav>
+                <div class="left-bar">
+                    <p class="time">9:41</p>
+                    <p class="name">Oscar Byiringiro</p>
+                </div>
+                <div class="right-bar">
+                    <div class="top-icons">
+                        <img src="./images/Cellular Connection.png">
+                        <img src="./images/Wifi.png">
+                        <img src="./images/Battery.png">
+                                    
+                                
+                    </div>
+                    <div class="bottom-icons">
+                        <img src="./images/Union.png">
+                            
+                    </div>
+                    
+                        
+                </div>
+            </nav>
+        </header>
+        <section class="main-body">
+            <div class="container">
+                <div class="intro">
+                    <h1>Hey There</h1>
+                    <h1>I'm Oscar</h1>
```

Comment:
- [ ] Please remove this section I circle red because that portion is not part of the design. It was added by the designer to the Figma design so that the design looks real, but in the real-life implementation of this project, that section is not inclusive. Please remove it.
![image](https://user-images.githubusercontent.com/84156714/212720684-acccfda6-9180-43d6-b20c-86e391fb69ee.png)